core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJVersion.scala

```diff
@@ -31,14 +33,12 @@ final case class IntelliJVersion(build: String, release: Option[String]) {
 object IntelliJVersion {
   implicit val configConvert: ConfigConvert[IntelliJVersion] = deriveConvert[IntelliJVersion]
 
-  // TODO(#253): replace Latest with reading from reference.conf
-  val Latest: IntelliJVersion = release("2021.2.1", "212.5080.55")
-
+  // ext = Some(configConvert.map(_.ext).toString) is a convenient way to receive `probe.intellij.version.ext` value
   def snapshot(build: String): IntelliJVersion = {
-    IntelliJVersion(build, None)
+    IntelliJVersion(build = build, release = None, ext = Some(configConvert.map(_.ext).toString))
```

Comment:
If you need to read base IJVersion from config, you can just use  something like `Latest.copy(build = build, release = release)` in both methods. And read `Latest` properly (from reference conf)